modules/menu/react/stories/stories.tsx

```diff
@@ -11,27 +11,29 @@ import {CanvasSystemIcon} from '@workday/design-assets-types';
 import Popper from '@material-ui/core/Popper';
 import {Transition} from 'react-transition-group';
 import ClickAwayListener from '@material-ui/core/ClickAwayListener';
-import {DropdownButton, ButtonProps, BetaButtonTypes} from '@workday/canvas-kit-react-button';
+import {Button, ButtonProps} from '@workday/canvas-kit-react-button';
+import {Card} from '@workday/canvas-kit-react-card';
+import {depth, spacing} from '@workday/canvas-kit-react-core';
 
 import Menu from '../lib/Menu';
 import MenuItem, {MenuItemProps} from '../lib/MenuItem';
 
 import README from '../README.md';
 
 const FocusableButton = React.forwardRef(
-  (props: ButtonProps<BetaButtonTypes>, ref: React.RefObject<HTMLButtonElement>) => (
-    <DropdownButton buttonRef={ref} {...props}>
+  (props: ButtonProps, ref: React.RefObject<HTMLButtonElement>) => (
+    <Button buttonRef={ref} {...props}>
```

Comment:
@vibdev Did we mean to remove `DropdownButton`? Here's what it looks like before and after:

Before:
https://workday.github.io/canvas-kit/?path=/story/menu--with-control-button
![Screen Shot 2019-08-06 at 12 08 05 PM](https://user-images.githubusercontent.com/338257/62564867-e4f31900-b842-11e9-99d6-c3d2d7208e37.png)

After:
![Screen Shot 2019-08-06 at 12 05 39 PM](https://user-images.githubusercontent.com/338257/62564883-ef151780-b842-11e9-93da-1e8a0b640e55.png)

@anicholls Can you confirm if this change is intentional?

Reply:
@jayscheidt was this something that got eaten by the cherry pick?

Reply:
Not sure if I can link the storybook from ghe, but the `After` screenshot matches what is in master of the old repo

Reply:
I wonder if @anicholls made the change when the lib moved?

Reply:
So the reason this was done was due to the lack of availability of the blue buttons in our product. They are coming, but not widely available yet. I forget who made the call, but either @jpante or Jesse Spencer requested these be changed back to the original orange style to more closely align with what's in product today for the short term. 

We don't have a dropdown button version in orange, so we just used the regular `Button`. With the blue buttons becoming available soon, I'd say we could probably keep this as `DropdownButton`.

Reply:
The clarification I got was to use the `<Button>` component (orange buttons) and we'll change as part of the the migration to the new blue buttons later. I'm approving and merging.